vger/src/lib.rs

```diff
@@ -418,26 +419,33 @@ impl Renderer for VgerRenderer {
         let transformed_y = transform[1] * pos.x + transform[3] * pos.y + transform[5];
         let pos = Point::new(transformed_x, transformed_y);
 
-        let coeffs = self.transform.as_coeffs();
-        let scale = (coeffs[0] + coeffs[3]) / 2. * self.scale;
+        let scale_x = transform[0];
+        let scale_y = transform[3];
+
+        let scale = (transform[0] + transform[3]) / 2. * self.scale;
+        if scale.abs() < 0.1 {
```

Comment:
I think this is fine. One thing that could be considered in the future is rendering rectangles in place of the text with the appropriate text color and spacing when the overall font size is less than a certain threshold.

Reply:
Ah that would be good. I'll add a comment/todo for now